R/java_download.R

```diff
@@ -1,7 +1,7 @@
 #' Download a Java distribution
 #'
-#' @param version The Java version to download. If not specified, defaults to the latest LTS version.
-#' @param distribution The Java distribution to download. If not specified, defaults to "Corretto".
+#' @param version `Integer` or `character` vector of length 1 for major version of Java to download or install. If not specified, defaults to the latest LTS version. Can be "8", "11", "17", "21", "22", or 8, 11, 17, 21, or 22.
```

Comment:
IMO it's better to be stricter here and only accept numbers, not strings.

Reply:
I thought of supporting specific java versions beyond major version at some point. Not sure it is worth it. But if it is, strings will be necessary.